__tests__/app.test.js

```diff
@@ -412,4 +412,31 @@ describe('DELETE/api/comments/:comment_id', () => {
       expect(response.body.msg).toBe("bad request")
     })
   });
+});
+
+describe('GET/api/users', () => {
+  test('STATUS - 200: should return an array of user objects', () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then((response) => {
+      console.log(response.body.users)
+      expect(response.body.users.length).toBe(4)
+      response.body.users.forEach((user) => {
+        expect(user).toMatchObject({
+          username: expect.any(String),
+          name: expect.any(String),
+          avatar_url: expect.any(String)
+        })
+      })
+    })
+  });
+  test('STATUS - 404: should return appropriate error when passed invalid endpoint', () => {
```

Comment:
This is a fine test but should be independent of this describe block. It only needs to be caught once.